model/nml/namelists.nml

```diff
@@ -292,19 +292,21 @@ $                           EXPFSPSI  : Activation of PSI scheme
 $                           EXPFSFCT  : Activation of FCT scheme
 $                           IMPFSN    : Activation of N implicit scheme
 $                           IMPTOTAL  : Activation of fully implicit scheme | Non splitting
-$                           EXPTOTAL  : Turn on implicit refraction (only with imptotal)
+$                           EXPTOTAL  : Activation of the Block explicit solver (e.g. parallel communication reduced to all directional components) 
```

Comment:
@aronroland I see this now. Apologies I was looking for a new variable for a new feature.  Did the implicit go away? Is this a T/F or something else? 

Should this also be updated in the inp file?

Reply:
No, the former comment was wrong

Reply:
yes, it must be updated in the inp file, we will do that

Reply:
... many thanks for your comments!

Reply:
I have added some more comments but as of the complexity I do not expect full clarification as long the manual is not updated

Reply:
Can you please add here that this is a T/F or some other value (it seems from the added regtest, this is a T/F and add this to the inp file?  I'm confused as to why knowing what happens if this value is true or false requires a write up in a manual.  Is there an issue already made for explaining this fully in the manual? If not, please add one.  Since the code is finally cleaned up otherwise, I'll run my first set of regression tests, once we get the inp/nml file updated with the EXPTOTAL, I'll ask others to start reviewing the PR again as well.